src/tests/ecalc_cli/snapshots/test_app/test_json_true/test_v3.json

```diff
@@ -616,6 +616,52 @@
           1.0
         ]
       },
+      "maximum_rate": {
+        "rate_type": "STREAM_DAY",
+        "regularity": [
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0
+        ],
+        "timesteps": [
+          "2020-01-01 00:00:00",
+          "2021-01-01 00:00:00",
+          "2022-01-01 00:00:00",
+          "2023-01-01 00:00:00",
+          "2024-01-01 00:00:00",
+          "2024-12-01 00:00:00",
+          "2026-01-01 00:00:00",
+          "2027-01-01 00:00:00",
+          "2028-01-01 00:00:00",
+          "2029-01-01 00:00:00",
+          "2030-01-01 00:00:00",
+          "2031-01-01 00:00:00"
+        ],
+        "unit": "Sm3/d",
+        "values": [
+          null,
```

Comment:
All snapshots show `null` as values for max rate. Expected? Are we able to find a real max rate that actually verifies the functionality?